hack/update-munge-docs.sh

```diff
@@ -21,6 +21,9 @@ set -o pipefail
 KUBE_ROOT=$(dirname "${BASH_SOURCE}")/..
 source "${KUBE_ROOT}/hack/lib/init.sh"
 
+git_upstream=$(kube::util::git_upstream_remote_name)
```

Comment:
I feel like we're really overloaded with globals in the scripts and it's not clear to me how or if they are used at all.  Given it isn't documented anywhere I'm inclined to beg forgiveness here and scrub it.  We can always add it back.